src/main/resources/native/linux/launcher.c

```diff
@@ -117,5 +117,12 @@ void getLibGLEShandle() {
 fprintf(stderr, "NOT IMPLEMENTED\n");
 }
 
+#ifdef GVM_17
```

Comment:
easier done with #ifndef

Reply:
I know, I was just following the same pattern as [here](https://github.com/gluonhq/substrate/blob/master/src/main/resources/native/android/c/launcher.c#L186), in case we ever need to add dummy methods for 17

Reply:
changed